doc/planning_scene_python/src/planning_scene_tutorial.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+import sys
+
+def main():
+    rospy.init_node("planning_scene_tutorial")
+    moveit_commander.roscpp_initialize(sys.argv)
+
+    # BEGIN_TUTORIAL
+    #
+    # Setup
+    # ^^^^^
+    #
+    # The :moveit_core_python:`core.planning_scene.PlanningScene` class can be easily setup and
+    # configured using a URDF and
+    # SRDF. This is, however, not the recommended way to instantiate a
+    # PlanningScene. At the time of writing there are not yet python bindings
+    # for the PlanningSceneMonitor, which is is the recommended method to
+    # create and maintain the current planning scene
+    # using data from the robot's joints and the sensors on the robot. In
+    # this tutorial, we will instantiate a PlanningScene class directly,
+    # but this method of instantiation is only intended for illustration.
+    import moveit_commander
+    import numpy as np
+    import rospy
```

Comment:
`rospy` and `moveit_commander` are used in lines 5 and 6, so it's too late to import them here.